$NetBSD: patch-configure,v 1.2 2019/03/11 13:46:12 jperkin Exp $

Don't automatically add libgcc on SunOS. Reported upstream as #75941.
Don't autodetect maintainer-zts.
The meta_ccld removal reported upstream as #75940.

--- configure.orig	2019-02-05 13:10:04.000000000 +0000
+++ configure
@@ -5453,9 +5453,6 @@ fi
 case $host_alias in
   *solaris*)
     CPPFLAGS="$CPPFLAGS -D_POSIX_PTHREAD_SEMANTICS"
-    if test "${enable_libgcc+set}" != "set" && test "$GCC" = "yes"; then
-      enable_libgcc=yes
-    fi
     ;;
   *dgux*)
     CPPFLAGS="$CPPFLAGS -D_BSD_TIMEOFDAY_FLAVOR"
@@ -6866,27 +6863,6 @@ EOF
     ;;
   esac
 
-  if test "$APACHE_VERSION" -lt 2004001; then
-    APXS_MPM=`$APXS -q MPM_NAME`
-    if test "$APXS_MPM" != "prefork" && test "$APXS_MPM" != "peruser" && test "$APXS_MPM" != "itk"; then
-
-  enable_maintainer_zts=yes
-  if test "$pthreads_working" != "yes"; then
-    as_fn_error $? "ZTS currently requires working POSIX threads. We were unable to verify that your system supports Pthreads." "$LINENO" 5
-  fi
-
-    fi
-  else
-    APACHE_THREADED_MPM=`$APXS_HTTPD -V | grep 'threaded:.*yes'`
-    if test -n "$APACHE_THREADED_MPM"; then
-
-  enable_maintainer_zts=yes
-  if test "$pthreads_working" != "yes"; then
-    as_fn_error $? "ZTS currently requires working POSIX threads. We were unable to verify that your system supports Pthreads." "$LINENO" 5
-  fi
-
-    fi
-  fi
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
 
@@ -83623,7 +83599,7 @@ $as_echo "#define HAVE_TIDYBUFFIO_H 1" >
   fi
 
   TIDY_LIBDIR=$TIDY_DIR/$PHP_LIBDIR
-  if test "$TIDY_LIB_NAME" == 'tidyp'; then
+  if test "$TIDY_LIB_NAME" = 'tidyp'; then
 
 $as_echo "#define HAVE_TIDYP_H 1" >>confdefs.h
 
